app/views/clients/viewclient.html

```diff
@@ -37,16 +37,31 @@ <h3 class="bolder">{{'label.button.deletesig' | translate}}</h3>
 		<div class="modal-header silver">
 			<img ng-src="{{largeImage}}" alt="Signature">
 		</div>
-		<div class="modal-body ">
+		<div class="modal-body">
```

Comment:
Once you make the changes in the controller, there won't be any need for the selecting template here, so you could delete it. Since clicking on the upload will navigate to the upload signature modal. Hope you understood :)